tests/test_state_scenario.py

```
@@ -1,16 +1,33 @@
-from galadriel.scenario.state import ScenarioState
+from galadriel.scenario.state import ScenarioState, AddScenarioState, EditScenarioState
 from galadriel.scenario.model import ScenarioModel, ScenarioCaseModel
 from conftest import init_state, LinkableEmptyCasesTests
 
 pytestmark = pytest.mark.integration
 
 
+@pytest.fixture(autouse=True)
+def mock_audit():
+    """Patch async user info lookup and log_action for all scenario tests."""
+    async def fake_get_user_info(self):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `fake_get_user_info`

(ANN202)